Real_Masters_all/whitepet.xml

```diff
@@ -557,7 +561,7 @@
               <unittitle>Private Financial Journal <unitdate type="inclusive" normal="1868/1878">1868-1878</unitdate></unittitle>
             </did>
             <odd>
-              <p>(out. vol. #15)</p>
+              <p>(oversize volume no. 15)</p>
```

Comment:
This may also be a good time to note that 'whitepet.xml' always makes me imagine that it's a finding aid about https://upload.wikimedia.org/wikipedia/commons/5/56/White_tiger-Gunma_Safari_Park.jpg